aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,56 @@
+import json
+import boto3
+
+
+def lambda_handler(event, context):
```

Ruff (0.8.2)

Missing return type annotation for public function `lambda_handler`

(ANN201)